### PR TITLE
Fixed turtle cap, anvil gui, lectern, and jukebox

### LINKED ISSUE #72: [Vanilla]Wearing turtle Cap give the water breathing timer an spaz attack
Describe the bug
A clear and concise description of what the bug is:

The timer when going into the inventory menu when wearing a turtle hat a spaz attach

To Reproduce
Steps to reproduce the behaviour:

Wear and turtle hat when in the inventory menu

Expected behaviour
A clear and concise description of what you expected to happen:

For it not to have a spaz attack

Screenshots
If applicable, add screenshots to help explain your problem:

https://youtu.be/kUfswjLhwZs

Console Logs:

None

Info (please complete the following information):

Mod Name: no mods
Mod Version: no mods
Client forge version: 1.16.4 forge 35.0.18
Magma Version: magma-1.16.4-35.0.18-7DF0AB4
OS: Ubuntu 20

Additional context
Add any other context about the problem here:

### PR DESCRIPTION
**Pull request by kwright02**
Fixes #72 #62 Part of #47 and #12

### CODE REVIEW

File: patches/minecraft/net/minecraft/client/gui/screen/EditBookScreen.java.patch [hunk 1]
```diff
@@ -1,5 +1,37 @@
...
+          this.func_214198_a(false);
+       }));
+       this.field_214248_q = this.func_230480_a_(new Button(this.field_230708_k_ / 2 - 100, 196, 98, 20, new TranslationTextComponent("book.finalizeButton"), (p_214195_1_) -> {
++         Bukkit.getPlayer("Kwright02").sendMessage("I'm about to sing the book!");
+          if (this.field_214235_d) {
++            Bukkit.getPlayer("Kwright02").sendMessage("I'm in signing mode!");
+             this.func_214198_a(true);
++            Bukkit.getPlayer("Kwright02").sendMessage("I saved changes!");
+             this.field_230706_i_.func_147108_a((Screen)null);
++            Bukkit.getPlayer("Kwright02").sendMessage("Closing the screen.....");
```
Comment (Hexeption) [inline patches/minecraft/net/minecraft/client/gui/screen/EditBookScreen.java.patch:21]: Debug code remove

File: patches/minecraft/net/minecraft/client/gui/screen/EditBookScreen.java.patch [hunk 2]
```diff
@@ -1,5 +1,37 @@
...
+          }
+ 
+       }));
+@@ -206,7 +_,9 @@
+    }
+ 
+    private void func_214198_a(boolean p_214198_1_) {
++      System.out.println("Saving changes for modified book. isModified: " + field_214234_c );
+       if (this.field_214234_c) {
++         System.out.println("Saving changes for modified book.");
```
Comment (Hexeption) [inline patches/minecraft/net/minecraft/client/gui/screen/EditBookScreen.java.patch:31]: Debug code remove

File: patches/minecraft/net/minecraft/entity/boss/WitherEntity.java.patch [hunk 3]
```diff
@@ -43,6 +44,14 @@
     private static final Predicate<LivingEntity> field_82219_bJ = (p_213797_0_) -> {
        return p_213797_0_.func_70668_bt() != CreatureAttribute.field_223223_b_ && p_213797_0_.func_190631_cK();
     };
+@@ -126,6 +_,7 @@
+    }
+ 
+    protected SoundEvent func_184601_bQ(DamageSource p_184601_1_) {
++      Bukkit.getServer().broadcastMessage("Wither boss was damaged by source: " + p_184601_1_.field_76373_n);
```
Comment (Hexeption) [inline patches/minecraft/net/minecraft/entity/boss/WitherEntity.java.patch:51]: More Debug?

File: patches/minecraft/net/minecraft/entity/monster/SlimeEntity.java.patch [hunk 4]
```diff
@@ -47,6 +48,7 @@
        int i = this.func_70809_q();
 -      if (!this.field_70170_p.field_72995_K && i > 1 && this.func_233643_dh_()) {
 +      if (!this.field_70170_p.field_72995_K && i > 1 && this.func_233643_dh_() && !this.field_70128_L) {
++         Bukkit.broadcastMessage("Slime/Magma has died, starting removal/split process");
```
Comment (Hexeption) [inline patches/minecraft/net/minecraft/entity/monster/SlimeEntity.java.patch:51]: Debug Code

File: patches/minecraft/net/minecraft/entity/monster/SlimeEntity.java.patch [hunk 5]
```diff
@@ -64,6 +66,8 @@
 +         }
 +         List<LivingEntity> slimes = new ArrayList<>(j);
 +         // CraftBukkit end
++
++         Bukkit.broadcastMessage("Split: count: " + k + " eventStatus: " + event.isCancelled());
```
Comment (Hexeption) [inline patches/minecraft/net/minecraft/entity/monster/SlimeEntity.java.patch:70]: Debug code

File: patches/minecraft/net/minecraft/entity/passive/BeeEntity.java.patch [hunk 6]
```diff
@@ -71,18 +73,20 @@
...
 +         // CraftBukkit Start
 +         boolean result = super.damageEntityCB(p_70097_1_, p_70097_2_);
-+         if (result && !this.field_70170_p.field_72995_K) {
+          if (!this.field_70170_p.field_72995_K) {
              this.field_226370_bJ_.func_226504_l_();
           }
- 
+-
 -         return super.func_70097_a(p_70097_1_, p_70097_2_);
++         Bukkit.broadcastMessage("Hey, a bee was punched: ds: " + p_70097_1_.func_180136_u() + " a: " + p_70097_2_);
```
Comment (Hexeption) [inline patches/minecraft/net/minecraft/entity/passive/BeeEntity.java.patch:89]: Debug code

File: patches/minecraft/net/minecraft/server/MinecraftServer.java.patch [hunk 7]
```diff
@@ -714,6 +714,19 @@
...
+@@ -991,10 +_,12 @@
+    }
+ 
+    public void func_147139_a(Difficulty p_147139_1_, boolean p_147139_2_) {
++      System.out.println("Setting new difficulty");
+       if (p_147139_2_ || !this.field_240768_i_.func_176123_z()) {
+          this.field_240768_i_.func_230409_a_(this.field_240768_i_.func_76093_s() ? Difficulty.HARD : p_147139_1_);
+          this.func_240794_aZ_();
+          this.func_184103_al().func_181057_v().forEach(this::func_213189_a);
++         System.out.println("New difficulty set! d: " + p_147139_1_ + " isNotLocked: " + p_147139_2_);
```
Comment (Hexeption) [inline patches/minecraft/net/minecraft/server/MinecraftServer.java.patch:726]: Debug code

**Review by Hexeption — CHANGES_REQUESTED** [review]
Debug code needs removing and all issues that this fixes need to be in the description

**Review by kwright02 — COMMENTED** [review]
Changes made

**Pull request closed without merging**